ZukoAzula2016/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
+encoding//src/org/team1540/zukoazula/RobotTemplate.java=UTF-8
```

Comment:
@amackworth colby included preferences in the first commit, so it kinda makes sense to include them. This also would stop the emoji breaking on @tytynguyen's computer I believe.